roles/installer/defaults/main.yml

```diff
@@ -5,3 +5,4 @@ foreman_installer_verbose: true
 foreman_installer_no_colors: false
 foreman_installer_options: []
 foreman_installer_locale: 'en_US.UTF-8'
+foreman_installer_timeout: 1800
```

Comment:
Can you add this to the documentation here -- https://github.com/theforeman/foreman-operations-collection/blob/develop/roles/installer/README.md?plain=1#L13